proto/cmp/types/v1alpha/seatmap.proto

```diff
@@ -0,0 +1,171 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha;
+
+import "cmp/types/v1alpha/description.proto";
+import "cmp/types/v1alpha/price.proto";
+
+// Basic representation of a seat with optional features.
+message Seat {
+  // Unique identifier, e.g., "12B", "A26"
+  string seat_id = 1;
+
+  // Seat type like STANDARD or PREMIUM etc
+  SeatType type = 2;
+
+  // Comma-separated features, e.g., "extra_leg_room,window"
+  // TODO: This field can be improved with a better enum and/or additional fields.
+  string features = 3;
+
+  // Min age for this seat
+  int32 min_age = 4;
+
+  // Max age for this seat. FIXME: Do we need this?
+  int32 max_age = 5;
+
```

Comment:
min age and max age are not the only conditions, so this should be a repeated string. For example for an emergency exit seat from southwest airlines:

Be 15 years of age or older
Have the capacity to perform the applicable functions without the assistance of an adult companion, parent, or other relative
Read and understand instructions related to emergency evacuation provided by Southwest in printed or graphic form
Understand oral crew commands
Have sufficient visual capacity to perform applicable functions without the assistance of visual aids beyond contact lenses or eyeglasses
Be able to hear and understand instructions shouted by Flight Attendants without assistance beyond a hearing aid
Adequately impart information orally in English to other passengers

Reply:
Updated the message types and the diagram above